package.json

```diff
@@ -41,7 +41,8 @@
     "debug": "^4.3.2",
     "immer": "9.0.5",
     "lodash": "^4.17.21",
-    "reduce-reducers": "^1.0.4"
+    "reduce-reducers": "^1.0.4",
+    "firebase": "^8.7.1"
```

Comment:
If we have another import then it will cause conflicts in the package deps. We already have `getFirestore()` function so we should get it from that instead. 
```ts
const serverTimestamp = (key) =>
  key === '::serverTimestamp' && getFirestore().FieldValue.serverTimestamp();

```